mpqp/execution/connection/google_connection.py

```diff
@@ -1,76 +1,28 @@
-from getpass import getpass
-from termcolor import colored
+from mpqp.execution.connection.env_manager import get_env_variable
 
-import cirq_ionq as ionq
-from cirq_ionq.ionq_exceptions import IonQException
-from mpqp.execution.connection.env_manager import get_env_variable, save_env_variable
 
-
-def test_ionq_connection() -> bool:
-    """
-    Test the connection to the IonQ service.
-
-    Returns:
-        bool: True if the connection is successful, False otherwise.
-    """
-    service = ionq.Service(default_target="simulator")
-    try:
-        service.list_jobs()
-        return True
-    except IonQException:
-        print(colored("Wrong credentials", "red"))
-        return False
-
-
-def config_ionq_account():
+def get_google_account_info() -> str:
```

Comment:
You should rename this function to get_ionq_account_info. However, it could be interesting to have a google_account_info with all the information about the configured devices, including IONQ and AQT here (because AQT is missing apparently)